modules/standard/Math.chpl

```diff
@@ -18,9 +18,10 @@
  */
 
 /*
-This module provides wrappers for <cmath> (math.h) numerical constants and
-routines.  Its symbols are provided by default; an explicit 'use' statement
-is not necessary.
+This module provides numerical constants and functions,
+including those found in <cmath> (math.h) headers.
```

Comment:
Do we want to guarantee all 100% inclusion of <cmath>? If not, we could go for "... including many of those found in <cmath> ..."